tools/cockpit.spec

```diff
@@ -376,8 +376,8 @@ This package contains the Cockpit shell UI assets.
 Summary: Cockpit user interface for storage, using Storaged
 Requires: %{name}-shell = %{version}-%{release}
 Requires: storaged >= 2.1.1
-Requires: storaged-lvm2 >= 2.1.1
-Requires: device-mapper-multipath
+Recommends: storaged-lvm2 >= 2.1.1
+Recommends: device-mapper-multipath
```

Comment:
It's sad but this needs to only happen on Fedora 24 and later. Previous versions of RPM did not support Recommends.